api/src/main/java/com/datatorrent/api/ControlTupleEnabledSink.java

```diff
@@ -18,7 +18,7 @@
  */
 package com.datatorrent.api;
 
-import org.apache.apex.api.UserDefinedControlTuple;
+import org.apache.apex.api.ControlTuple;
```

Comment:
It should be equally simple and straightforward for applications developers and community (platform/operators developers). My preference is to distinguish control tuples aka markers such as BW, EW, CHK from user/operator defined tuples. A simple ControlTuple name may lead to a confusion. It may be a class name or a package name that allows distinguishing user/operator defined tuples from the platform control tuples.